internal/hooks.go

```diff
@@ -0,0 +1,42 @@
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+func RunHook(wd, hookName, pgHost string, pgPort int, pgUser, pgPassword, pgDatabase, pgSSLMode string) error {
+	filePath := filepath.Join(wd, "hooks", fmt.Sprintf("%s.sh", hookName))
+	if _, err := os.Stat(filePath); errors.Is(err, os.ErrNotExist) {
+		log.Printf("Skipping hook %s", hookName)
+
+		return nil
+	}
+
+	log.Printf("Running hook %s", hookName)
+
+	cmd := exec.Command(filePath)
+	cmd.Env = append(
+		os.Environ(),
+		fmt.Sprintf("PGHOST=%s", pgHost),
+		fmt.Sprintf("PGPORT=%d", pgPort),
+		fmt.Sprintf("PGUSER=%s", pgUser),
+		fmt.Sprintf("PGPASSWORD=%s", pgPassword),
+		fmt.Sprintf("PGDATABASE=%s", pgDatabase),
+		fmt.Sprintf("PGSSLMODE=%s", pgSSLMode),
```

Comment:
they already exist inside `TREK_` which is supplied to the container. i did this for simplicity reasons, so that the scripts can just call `psql` without any options, but your suggestions makes sense. i'll just drop this.